plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotationDescriptor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.erd.ui.notations;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.jkiss.dbeaver.model.impl.AbstractDescriptor;
+
+public class ERDNotationDescriptor extends AbstractDescriptor {
+
+    private static final String EXT_ATTRIBUTE_ID = "id"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_NAME = "name"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_DESCRIPTION = "description"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_DEFAULT = "isDefault"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_NOTATION = "notation"; // $NON-NLS-N$
+
+    private String id;
+    private String name;
+    private String description;
+    private boolean isDefault = false;
+    private ERDNotation notation;
+
+    protected ERDNotationDescriptor(IConfigurationElement cf) throws CoreException {
+        super(cf);
+        this.id = cf.getAttribute(EXT_ATTRIBUTE_ID);
+        this.name = cf.getAttribute(EXT_ATTRIBUTE_NAME);
+        this.description = cf.getAttribute(EXT_ATTRIBUTE_DESCRIPTION);
+        this.isDefault = Boolean.valueOf(cf.getAttribute(EXT_ATTRIBUTE_DEFAULT));
+        this.notation = (ERDNotation) cf.createExecutableExtension(EXT_ATTRIBUTE_NOTATION);
```

Comment:
Please use `ObjectType` instead of directly initializing it here.

Reply:
Please see the example implementation from parallel class
```
 public ERDExportFormatHandler getInstance() throws DBException {
            return type.createInstance(ERDExportFormatHandler.class);
        }
```
During the first call of registry (getInstance()) the instance of extension class created, no Lazy instantiating here.